### PR TITLE
Pretty print statements

### PR DESCRIPTION
**Pull request by turbolent**
Work towards #209

## Description

Implement pretty-printing functions for all statement AST nodes:

- `ReturnStatement`
- `BreakStatement`
- `ContinueStatement`
- `IfStatement`
- `WhileStatement`
- `ForStatement`
- `EmitStatement`
- `AssignmentStatement`
- `SwapStatement`
- `ExpressionStatement`

Also:
- Add missing JSON marshaling tests
- Allow inserting tabs (4 spaces) in the pretty printer tool's text area

______

- [ ] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: runtime/ast/statement.go [hunk 1]
```diff
@@ -251,6 +331,36 @@ func (s *ForStatement) EndPosition() Position {
 	return s.Block.EndPosition()
 }
 
+const forStatementForKeywordSpaceDoc = prettier.Text("for ")
+const forStatementSpaceInKeywordSpaceDoc = prettier.Text(" in ")
+
+func (s *ForStatement) Doc() prettier.Doc {
+	doc := prettier.Concat{
```
Comment (SupunS) [inline runtime/ast/statement.go:338]: need to concat?

File: runtime/ast/statement.go [hunk 2]
```diff
@@ -251,6 +331,36 @@ func (s *ForStatement) EndPosition() Position {
...
+		forStatementForKeywordSpaceDoc,
+	}
+
+	if s.Index != nil {
+		doc = append(
```
Comment (SupunS) [inline runtime/ast/statement.go:343]: Is there a difference between using an array (append) vs `prettier.Concat{}`

**Review by SupunS — APPROVED** [review]
Nice!

File: runtime/ast/statement.go (hunk 1 above)
Comment (turbolent) [reply runtime/ast/statement.go:338]: Yes, we always need the concatenation, as https://github.com/onflow/cadence/pull/1280/files#diff-cb6a372e592ff206bc8f5ad13e85ba6527d1e6914f281f0dd263479c7f3a278eR350 always appends more after the conditionally appended index (https://github.com/onflow/cadence/pull/1280/files#diff-cb6a372e592ff206bc8f5ad13e85ba6527d1e6914f281f0dd263479c7f3a278eR342)

File: runtime/ast/statement.go (hunk 2 above)
Comment (turbolent) [reply runtime/ast/statement.go:343]: `prettier.Concat` is a prettier document, but is also just a slice of documents, so we can use the Go `append` to add to it

**Pull request merged**